saleor/webhook/tests/test_webhook_payloads.py

```diff
@@ -183,3 +183,21 @@ def test_generate_product_variant_deleted_payload(
     assert len(payload["attributes"]) == 2
     assert len(payload["channel_listings"]) == 1
     assert len(payload.keys()) == len(payload_fields)
+
+
+def test_generate_invoice_payload(fulfilled_order):
+    invoice = fulfilled_order.invoices.first()
+    payload = json.loads(generate_invoice_payload(invoice))[0]
+    order = payload["order"]
+
+    assert set(payload.keys()) == {
+        "id",
+        "number",
+        "external_url",
+        "created",
+        "order",
+        "type",
+    }
+    assert set(order.keys()).issuperset(ORDER_FIELDS)
```

Comment:
In my opinion, we should check values instead of checking only keys in the payload.